telemetry/service/service-model.json

```diff
@@ -1,205 +1,236 @@
 {
-    "version": "2.0",
-    "metadata": {
-        "apiVersion": "2017-07-25",
-        "endpointPrefix": "client-telemetry",
```

Comment:
This changed from `client-telemetry` to `ToolkitTelemetry` -- did that change the generated C# code?

Reply:
it changes the "RegionEndpointServiceName" property, but since we're not in the endpoints.json, i don't think it matters, but reverted just in case it does